docs/_pages/playbooks.md

```diff
@@ -10,6 +10,8 @@ sidebar:
 
 | Name    | Detections | Type        |
 | --------| ---------- | ----------- |
+| [AWS Disable User Accounts](/playbooks/aws_disable_user_accounts/)| None | Input |
```

Comment:
There seem to be two kinds of "types". There is input vs automation, and investigation vs response. To be consistent, this one should be "Response" I think.